feature/setting/src/main/java/com/boostcamp/dreamteam/dreamdiary/setting/SettingBackupScreen.kt

```diff
@@ -0,0 +1,97 @@
+package com.boostcamp.dreamteam.dreamdiary.setting
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material.icons.outlined.Alarm
+import androidx.compose.material.icons.outlined.CloudDownload
+import androidx.compose.material.icons.outlined.CloudUpload
+import androidx.compose.material.icons.outlined.Info
+import androidx.compose.material.icons.outlined.Restore
+import androidx.compose.material.icons.outlined.Sync
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.boostcamp.dreamteam.dreamdiary.designsystem.theme.DreamdiaryTheme
+import com.boostcamp.dreamteam.dreamdiary.setting.component.SettingCategory
+import com.boostcamp.dreamteam.dreamdiary.setting.component.SettingOption
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+internal fun SettingBackupScreen(onBackClick: () -> Unit) {
+    val rememberScrollState = rememberScrollState()
```

Comment:
다른 사용처가 없는 것 같은데 따로 만드신 이유가 있으신가요?

Reply:
SettingBackupScreen 전체를 말하는 것인가요???

설정(SettingScreen) 에 있는 데이터 `백업 및 복원`을 클릭할 때 이동하는 화면 입니다

Reply:
```kt
        Column(
            modifier = Modifier
                .padding(innerPadding)
                .fillMaxWidth()
                .padding(8.dp)
                .verticalScroll(rememberScrollState),
        ) 
```

요기 사용하고 있습니다

Reply:
그러면 저기에 바로 넣어두어도 되지 않나요?

```suggestion
        Column(
            modifier = Modifier
                .padding(innerPadding)
                .fillMaxWidth()
                .padding(8.dp)
                .verticalScroll(rememberScrollState()),
        ) 
```

Reply:
ㅎㅎㅎ